frontend/src/store/facilityBooking/action.ts

```diff
@@ -211,6 +268,16 @@ export const editBookingDescription = (newBookingDescription: string) => (dispat
   dispatch({ type: FACILITY_ACTIONS.SET_BOOKING_DESCRIPTION, newBookingDescription: newBookingDescription })
 }
 
+/**
+ *
+ * Takes in a new date object and updates both start and end dates to that date.
+ *
+ * @params newDate (date)
```

Comment:
nice catch, `Date` is capital letter ;)
```suggestion
 * @params newDate (Date)
```